packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -9,6 +9,8 @@ import { randInt } from '../../lib/random';
 import SimpleSchema from 'simpl-schema';
 import Button from '@material-ui/core/Button';
 import Input from '@material-ui/core/Input';
+import MailOutline from '@material-ui/icons/MailOutline'
```

Comment:
I added the mail icon to the EA Forum version only, since I thought it might not fit in with LW